examples/terraform/nutanix/main.tf

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright 2022 The KubeOne Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+provider "nutanix" {
+  insecure = true
+}
+
+data "nutanix_cluster" "cluster" {
+  name = var.nutanix_cluster_name
+}
+
+data "nutanix_project" "project" {
+  project_id = var.project_id
+}
+
+data "nutanix_subnet" "subnet" {
+  subnet_name = var.subnet_name
+}
+
+data "nutanix_image" "image" {
+  image_name = var.image_name
+}
+
+resource "nutanix_category_key" "category_key" {
+  name        = "KubeOne-Cluster"
```

Comment:
Should be `KubeOneCluster` as far as I can say. The built-in categories are all camel case and I was planning to add `KKPCluster` as category key (but I'm very much open to discussing this).

Btw, will this work with multiple cluster?